Regression/PrimevalTitmouse/Regression.cs

```diff
@@ -321,7 +321,7 @@ private void ReceiveMouseChanged(object sender, ButtonPressedEventArgs e)
             }
 
             //Handle a Right Click
-            if (e.Button == SButton.MouseRight)
+            if (e.Button == SButton.MouseLeft)
```

Comment:
What necessitated this change?

Reply:
I have changed it. Because when you are holding an item and, for example, trying to open a chest, you will put it on. So I changed it to Left button.